expo/gbe/forms.py

```diff
@@ -27,8 +27,12 @@ class ParticipantForm(forms.ModelForm):
     required_css_class = 'required'
     error_css_class = 'error'
     email = forms.EmailField(required=True)
-    first_name = forms.CharField(required=True)
-    last_name = forms.CharField(required=True)
+    first_name = forms.CharField(required=True,
+                         label=participant_labels['legal_first_name'],
+                         help_text=participant_form_help_texts['legal_name'])
+    last_name = forms.CharField(required=True,
+                        label=participant_labels['legal_last_name'],
+                        help_text=participant_form_help_texts['legal_name'])
```

Comment:
check for pep8 on these:

forms.py:31:26: E128 continuation line under-indented for visual indent
forms.py:32:26: E128 continuation line under-indented for visual indent
forms.py:34:25: E128 continuation line under-indented for visual indent
forms.py:35:25: E128 continuation line under-indented for visual indent